### PR TITLE
Updated AI docs and image

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/configuration/ai-analysis.rst [hunk 1]
```diff
@@ -7,18 +7,12 @@ Robusta can integrate with `Holmes GPT <https://github.com/robusta-dev/holmesgpt
...
 
-When enabled, your Slack messages will have the ``Ask Holmes`` button:
-
-   .. image:: /images/pod-pending-alert.png
-       :width: 600px
-
-Clicking it, will launch a Holmes investigation, which will send the results back to Slack:
-
-   .. image:: /images/pending-alert-holmes-response.png
+   .. image:: /images/robusta-holmes-investigation.png
```
Comment (aantn) [inline docs/configuration/ai-analysis.rst:12]: Do you have an indentation here which leads to the image being block quoted?

Screenshot 2024-07-01 at 11 13 43

File: docs/configuration/ai-analysis.rst [hunk 2]
```diff
@@ -7,18 +7,12 @@ Robusta can integrate with `Holmes GPT <https://github.com/robusta-dev/holmesgpt
...
        :width: 600px
 
+Clicking it, will launch a Holmes investigation and the results will be sent back to Slack or Robusta UI.
```
Comment (aantn) [inline docs/configuration/ai-analysis.rst:15]: The UI part is confusing. Clicking that button does not send results to the Robusta UI. 

Can you document the UI part separately?

**Review by aantn — APPROVED** [review]

File: docs/configuration/ai-analysis.rst (hunk 1 above)
Comment (pavangudiwada) [reply docs/configuration/ai-analysis.rst]: Fixed

File: docs/configuration/ai-analysis.rst (hunk 2 above)
Comment (pavangudiwada) [reply docs/configuration/ai-analysis.rst]: Yes documented it
image

**Comment by aantn** [comment]
LGTM

**Pull request merged**